Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -20,8 +21,8 @@
 
 	double pulsesPerRevolution = 1440.0;
 	double averageDistance;
-	// WPI_TalonSRX leftBackMotor;
-	// WPI_TalonSRX rightBackMotor;;
+	WPI_TalonSRX leftBackMotor;
+	WPI_TalonSRX rightBackMotor;;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)